### PR TITLE
Confirm pt autotranslated definitions.

### PR DESCRIPTION
**Pull request by dlyongemallo**

### CODE REVIEW

**Comment by De7vID** [comment]
@thiagocmatos Are these good? Any edits?

File: mem-28-examples.xml [hunk 1]
```diff
@@ -807,7 +807,7 @@ Jos haluat vastata kieltävästi, sano {Qo':excl} "ei, kieltäydyn".</column>
       <column name="definition_sv">Förbannelse-krigföring har definitivt börjat. [AUTOTRANSLATED]</column>
       <column name="definition_ru">Война проклятий определенно началась. [AUTOTRANSLATED]</column>
       <column name="definition_zh_HK">詛咒戰爭肯定已經開始。 [AUTOTRANSLATED]</column>
-      <column name="definition_pt">A guerra de maldições definitivamente começou. [AUTOTRANSLATED]</column>
+      <column name="definition_pt">A guerra de maldições definitivamente começou.</column>
```
Comment (thiagocmatos) [inline mem-28-examples.xml:810]: Better update to:
A guerra de insultos definitivamente começou.

File: mem-28-examples.xml [hunk 2]
```diff
@@ -3182,7 +3182,7 @@ The lyrics are by Lawrence Schoen, and the accompanying music is by Rich Yampell
       <column name="definition_sv">Avskedad! [AUTOTRANSLATED]</column>
       <column name="definition_ru">Уволен! [AUTOTRANSLATED]</column>
       <column name="definition_zh_HK">解散！ [AUTOTRANSLATED]</column>
-      <column name="definition_pt">Demitido! [AUTOTRANSLATED]</column>
+      <column name="definition_pt">Demitido!</column>
```
Comment (thiagocmatos) [inline mem-28-examples.xml:3185]: Better change to:
Dispensado!

**Review by thiagocmatos — CHANGES_REQUESTED** [review]
I have proposed two updates.

**Pull request merged**